src/server/handlers/metrics.py

```diff
@@ -53,7 +54,11 @@ def _retrieve_by_kwargs(self, table_name, database, schema):
                     Metric.table_name,
                     Metric.database,
                     Metric.schema,
-                    func.max(Metric.created_at)
+                    func.max(Metric.created_at),
+                    ZScore.error,
+                ).join(
+                    ZScore,
+                    ZScore.id == ZScore.metric_id,
```

Comment:
Is this right or should it be `Metric.id` ?

Reply:
You right. thanks